typescript-functions/main.ts

```diff
@@ -0,0 +1,45 @@
+function convertMinutesToSeconds(minutes: number): number {
+  return minutes * 60;
+}
+console.log(convertMinutesToSeconds(5));
+
+function greet(name: string): string {
+  return 'Hi' + ' ' + name;
+}
+console.log(greet('chubbybunny'));
+
+const getArea = (width: number, height: number): number => {
+  return width * height;
+};
+console.log(getArea(17, 42));
+
+const getFirstName = (person: {
```

Comment:
If you move your `interface Person` above this, you can use the `person: Person` here instead of writing out the object again to save of code and improve readability